packages/components/preview/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>packages/components</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1"><script defer src="/goodhood/packages/components/preview/main.js"></script><link href="/goodhood/packages/components/preview/main.css" rel="stylesheet"></head>
+  <body>
+  </body>
+</html>
```

Comment:
I don't get why suddenly it's required to include a preview folder in the components package.
And as far as I understand it's not even used. Instead the root preview folder is used.

Reply:
I think @iamuchejude accidentally commited the preview folder generated by `npm run tryBuild:preview`